README.rst

```diff
@@ -798,14 +798,27 @@ Choose from the list of available rules:
 * **multiline_whitespace_before_semicolons**
 
   Forbid multi-line whitespace before the closing semicolon or move the
-  semicolon to the new line for chained calls.
+  semicolon to the new line for chained calls. DEPRECATED: use
+  ``multiline_whitespace_before_statement_end`` instead.
 
   Configuration options:
 
   - ``strategy`` (``'new_line_for_chained_calls'``, ``'no_multi_line'``): forbid
     multi-line whitespace or move the semicolon to the new line for chained
     calls; defaults to ``'no_multi_line'``
 
+* **multiline_whitespace_before_statement_end**
+
+  Forbid multi-line whitespace before a statement end (comma or semicolon)
+  or moves it to the next line for chained calls.
+
+  Configuration options:
+
+  - ``comma_strategy`` (``'new_line_for_chained_calls'``, ``'no_new_line'``, ``'none'``):
+    strategy to apply to comma; defaults to ``'no_new_line'``
+  - ``semicolon_strategy`` (``'new_line_for_chained_calls'``, ``'no_new_line'``,
+    ``'none'``): strategy to apply to semicolon; defaults to ``'no_new_line'``
```

Comment:
for minimize diff size,
we like to have trailing comma and semicolons on next line.

Can we reflect that in default settings of this new rule, please?
```
            'comma_strategy' => 'no_new_line',
            'semicolon_strategy' => 'new_line_for_chained_calls',
```

Then, please add rule to local `.php_cs.dist` file

Reply:
Did you mean
```php
            'comma_strategy' => 'new_line_for_chained_calls',
            'semicolon_strategy' => 'new_line_for_chained_calls',
```
?

Reply:
i got confused by multi vs single...

yes, I would opt for the `new_line_for_chained_calls` x2 as defaults, as it minimizes diff size

Reply:
Done.